lib/page/login_page.dart

```diff
@@ -76,98 +82,117 @@ class _LoginPageState extends State<LoginPage> {
-      padding: EdgeInsets.only(left: 10,right: 10),
       decoration: BoxDecoration(
           color: Colors.white,
-          borderRadius: BorderRadius.circular(5)
-      ),
+          border:
+              Border(bottom: BorderSide(width: 0.3, color: Color(0xff2A2A2A)))),
       child: pwdField,
     );
   }
 
   _getLogin(BuildContext context) {
-    return  Container(
-      margin: EdgeInsets.only(left: 60,right: 60),
+    return Container(
+      margin: EdgeInsets.only(left: 30, right: 30),
       height: 50,
       width: MediaQuery.of(context).size.width,
       child: RaisedButton(
-        onPressed: (){
-          if(null != account && account.length > 0 && null != pwd && pwd.length > 0){
+        onPressed: () {
+          if (null != account &&
+              account.length > 0 &&
+              null != pwd &&
+              pwd.length > 0) {
             loginController.login(account, pwd);
-          }else{
+          } else {
             EasyLoading.showToast('Check your info and try again.');
           }
         },
-        child: Text('Log in',style: TextStyle(color: Colors.white,fontSize: 20),),
-        color: ColorRes.color_3,
-        shape: RoundedRectangleBorder(
-            borderRadius:BorderRadius.circular(5)
+        child: Text(
+          'Log in',
+          style: TextStyle(color: Colors.white, fontSize: 20),
         ),
+        color: ColorRes.color_3,
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(5)),
       ),
     );
   }
 
   _getBottomLayout(BuildContext context) {
     return Container(
       width: MediaQuery.of(context).size.width,
-      margin: EdgeInsets.only(left: 60,right: 60),
+      margin: EdgeInsets.only(left: 25, right: 30),
       height: 40,
       child: Stack(
         children: [
           Positioned(
               left: 0,
               child: TextButton(
-                onPressed: (){
+                onPressed: () {
                   EasyLoading.showToast('功能待开发');
                 },
-                child: Text('Forget password',style: TextStyle(color: Colors.white),),
-              )),
-          Positioned(
-              right: 0,
-              child: TextButton(
-                onPressed: (){
-                  Get.toNamed(Routers.register);
-                },
-                child: Text('Sign up',style: TextStyle(color: Colors.white),),
+                child: Text(
+                  'Forgot password?',
+                  style: TextStyle(color: Color(0xff2A2A2A), fontSize: 13),
+                ),
               )),
+          // TODO: 暂时移除注册功能
+          // Positioned(
+          //     right: 0,
+          //     child: TextButton(
+          //       onPressed: () {
+          //         Get.toNamed(Routers.register);
+          //       },
+          //       child: Text(
+          //         'Sign up',
+          //         style: TextStyle(color: Colors.black),
+          //       ),
+          //     )),
```

Comment:
This code should not commit to pull request. You can keep them in your local only.